docs/contributor_guide.rst

```diff
@@ -102,9 +115,23 @@ Enforcing code style
 
 .. _prtag:
 
-PR title tags
--------------
-Please always prepend exactly one tag such as ``[Metal]`` to PR titles. For example, "[Metal] Support bitmasked SNode", "[OpenGL] AtomicMin/Max support", or "[Opt] Enhanced constant folding".
+PR title format and tags
+------------------------
+PR titles will be part of the commit history reflected in the ``master`` branch, therefore it is important to keep PR titles readable.
+
+ - The first letter of the PR title body should be capitalized, unless the title starts with an identifier;
+
+     - E.g., "[doc] improve documentation" should be formatted as "[doc] Improve documentation";
+     - "[Lang] ``ti.sqr(x)`` is now deprecated" is fine because ``ti`` is an identifier.
+
+ - Please always prepend at least one tag such as ``[Metal]`` to PR titles;
+
+     - When using multiple tags, make sure there is exactly one space between tags;
+     - E.g., "[Metal][refactor]" (no space) should be formatted as "[Metal] [refactor]";
+
+ - Please do not include back quotes ("`") in PR titles.
```

Comment:
Let's use \\\` in `misc/make_changelog.py` then, having back quotes are very necessary usages like \`\`ti.sqr(x)\`\`.

Reply:
A raw ti.sqr(x) is not too bad. Let's simply avoid `'s. This simplifies everything.